### PR TITLE
Add git-merge and slide on merge conflics

### PR DESCRIPTION
**Pull request by martialblog**
- day1: Adds a new Lab in git branch section with a git merge
 - day2: Adds a Slide explaining Merge Conflics before we simulate one

See #99

### CODE REVIEW

**Comment by martialblog** [comment]
Ironically this will likely create a Merge Conflict with the mermaid diagrams Branch...

**Comment by martialblog** [comment]
@htriem @mocdaniel Feedback is welcome

File: day2/01_Workflows/02_Centralized.md [hunk 1]
```diff
@@ -71,6 +71,28 @@ for integrating and sharing code changes between repositories.
...
 
+~~~ENDSECTION~~~
+
+!SLIDE smbullets
+# Centralized Workflow - Merge Conflicts
+
+* Conflicts arise when the same lines in a file have changed
+* When merging Git will try to automatically integrate new changes
+* Git cannot automatically determine what is correct
+* `>>>` marks conflicts and show the differences
```
Comment (theFeu) [inline day2/01_Workflows/02_Centralized.md:82]: * `===` is the barrier between the two states

Hint: those markers need to be deleted from the files

**Review by theFeu — CHANGES_REQUESTED** [review]
I would add this info to the file :)

File: day2/01_Workflows/02_Centralized.md (hunk 1 above)
Comment (martialblog) [reply day2/01_Workflows/02_Centralized.md:82]: Good idea

Comment (martialblog) [reply day2/01_Workflows/02_Centralized.md]: Done

**Pull request merged**